### PR TITLE
Actually execute slide out in Traverse action

### PR DESCRIPTION
**Pull request by PawelLipski**
😅 that's embarrassing (my fault)

image

### CODE REVIEW

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/TraverseSyncToParent.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ private void handleMergedToParent(
         // Let's just go straight to the next branch.
         Runnable doInUIThreadWhenReady = () -> graphTable.queueRepositoryUpdateAndModelRefresh(traverseNextEntry);
         new SlideOutBackgroundable(getString("action.GitMachete.BaseSlideOutAction.task.title"),
-            managedBranch, gitRepository, currentBranchIfManaged, branchLayout, graphTable, doInUIThreadWhenReady);
+            managedBranch, gitRepository, currentBranchIfManaged, branchLayout, graphTable, doInUIThreadWhenReady).queue();
```
Comment (PawelLipski) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/TraverseSyncToParent.java:129]: :trollface:

Comment (mkondratek) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/TraverseSyncToParent.java:129]: XD

File: qual/src/main/java/com/virtuslab/qual/async/BackgroundableQueuedElsewhere.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * Such cases are detected by ArchUnit test {@code com.virtuslab.archunit.BackgroundTaskEnqueuingTestSuite}.
+ * Still, in some rare cases it might happen that a method passes the newly-created backgroundable
+ * to downstream APIs to actually enqueue.
+ * Such methods must be marked as {@link BackgroundableQueuedElsewhere} for {@code BackgroundTaskEnqueuingTestSuite} to ignore.
+ * <p>
+ * This annotation must have runtime retention to be visible to ArchUnit tests.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD})
+public @interface BackgroundableQueuedElsewhere {}
```
Comment (mkondratek) [inline qual/src/main/java/com/virtuslab/qual/async/BackgroundableQueuedElsewhere.java:22]: xD

File: qual/src/main/java/com/virtuslab/qual/async/BackgroundableQueuedElsewhere.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Used to mark methods that create an instance of {@code com.intellij.openapi.progress.Task.Backgroundable},
+ * but do <b>not</b> call {@code #queue()}.
+ * <p>
+ * Typically, it is a smell that indicates that the programmer forgot to actually schedule the task.
```
Comment (mkondratek) [inline qual/src/main/java/com/virtuslab/qual/async/BackgroundableQueuedElsewhere.java:12]: 💟

**Review by mkondratek — APPROVED** [review]

**Pull request merged**